Provisioner/BuildProvisioner.php

```diff
@@ -271,14 +265,18 @@ protected function createCrontab(SSH2 $ssh, ApplicationEnvironment $applicationE
 
             $this->executeSshCommand($ssh, $command);
 
-            $this->taskService->addSuccessLogMessage($this->task, 'Crontab created.', 2);
+            $this->taskLoggerService->addSuccessLogMessage($this->task, 'Crontab created.', 2);
         } catch (\Exception $ex) {
-            $this->taskService
+            $this->taskLoggerService
                 ->addErrorLogMessage($this->task, $ex->getMessage(), 2)
                 ->addFailedLogMessage($this->task, 'Creating crontab failed.', 2);
 
             throw $ex;
```

Comment:
LoggedException

Reply:
This exception is caught in the `doRun` method and re-thrown as a LoggedException there.